validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -161,7 +161,7 @@ public OkHttpValidatorRestApiClient(final HttpUrl baseEndpoint, final OkHttpClie
             Map.of("slot", slot.toString()),
             queryParams,
             createHandler(GetNewBlockResponse.class))
-        .map(response -> response.data);
+        .map(response -> (BeaconBlock) response.data);
```

Comment:
Can we get away with:
```suggestion
        .<BeaconBlock>map(response -> response.data);
```

Where possible it's better to just be explicit about generics rather than using a cast.